templates/AureliaSpa/ClientApp/app/components/app/app.ts

```diff
@@ -0,0 +1,19 @@
+import {Aurelia} from 'aurelia-framework';
+import {Router, RouterConfiguration} from 'aurelia-router';
+
+export class App {
+  router: Router;
+
+  configureRouter(config: RouterConfiguration, router: Router) {
+    config.title = 'Aurelia';
+    config.map([
+      { route: ['', 'home'], name: 'home', settings:{class:'home'},     moduleId: '../home/home',  nav: true, title: 'Home' },
```

Comment:
The use of the term `class` here is a bit odd. I'd naturally think that refers to a class in the code, but it looks like you're using that to mean "CSS class we should set on the icon element". Could you rename this to something more descriptive, such as `icon` or `icon-className`?